movieReviewApp/src/main/java/com/mobiquity/movieReviewApp/service/WatchlistService.java

```diff
@@ -0,0 +1,13 @@
+package com.mobiquity.movieReviewApp.service;
+
+public interface WatchlistService {
+
+  String addMovieToUserWatchlist(String emailId, String movieName);
```

Comment:
maybe change the names of this to addMovie(), removeMovie(), addSeries(), removeSeries(). I think it's redundant adding the WatchList at the end and it makes it less readable to me.

Reply:
good point